### PR TITLE
feat: Enhancements to page

### PR DESCRIPTION
**Pull request by vladislavkeblysh**
- Fixed font family on page 
- Fixed borders on action bar on mobile view
- Fixed newest first text overlap another text
- Fixed displaying user name on mobile view

Screenshot 2023-10-12 at 15 07 45
Screenshot 2023-10-12 at 15 07 18
Screenshot 2023-10-12 at 15 08 31
Screenshot 2023-10-12 at 15 08 15
Screenshot 2023-10-12 at 15 09 05
Screenshot 2023-10-12 at 15 08 51
Screenshot 2023-10-12 at 15 10 31
Screenshot 2023-10-12 at 15 10 14

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @vladislavkeblysh!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by itsjeyd** [comment]
@vladislavkeblysh Thank you for this contribution!

@jmakowski1123 With this set of changes I'm not sure if we need to step through the full product review process. It seems like they are just fixing some small visual inconsistencies. What do you think?

* Palm backport: #576 
* Quince backport: #578 

CC @mphilbrick211

**Comment by itsjeyd** [comment]
@jmakowski1123 Friendly ping about [this question](https://github.com/openedx/frontend-app-discussions/pull/577#issuecomment-1766566787) -- do you think this PR needs a round of product review?

File: src/index.scss [hunk 1]
```diff
@@ -347,9 +347,17 @@ header {
...
+          @media screen and (max-width: 567px) {
+            border-right: solid 1px #e9e6e4;
+          }
       }
     }
   }
+
+  button {
+    font-family: inherit;
+  }
```
Comment (sundasnoreen12) [inline src/index.scss:360]: Once the theme is applied, the font-family issue will be resolved. so we don't need to add this css.
`npm install --no-save "@edx/brand@git+https://[git@github.com](mailto:git@github.com)/edx/brand-edx.org#master"`

File: src/index.scss [hunk 2]
```diff
@@ -479,7 +487,7 @@ header {
 }
 
 .font-style {
-  font-family: "Inter";
+  font-family: inherit;
```
Comment (sundasnoreen12) [inline src/index.scss:490]: This change should also be reverted as it will be resolved once the theme is applied.

**Review by sundasnoreen12 — CHANGES_REQUESTED** [review]

File: src/index.scss (hunk 2 above)
Comment (awais-ansari) [reply src/index.scss:490]: This issue has been resolved and merged. Please rebase this branch.

File: src/index.scss (hunk 1 above)
Comment (awais-ansari) [reply src/index.scss:360]: @sundasnoreen12 This theme is related to edx.

Comment (awais-ansari) [reply src/index.scss:360]: @vladislavkeblysh Font-family issue has been resolved in master branch so we do not need this change.

**Comment by itsjeyd** [comment]
@jmakowski1123 @awais-ansari It [sounds like](https://github.com/openedx/frontend-app-discussions/pull/576#issuecomment-1801949759) product review for this PR has already started (?). If so, could you please point me to where that conversation is happening? It doesn't seem like we have a feature ticket for this PR, so it's hard for me to tell what its current status is.

CC @mphilbrick211

File: src/index.scss (hunk 2 above)
Comment (vladislavkeblysh) [reply src/index.scss]: Fixed

File: src/index.scss (hunk 1 above)
Comment (vladislavkeblysh) [reply src/index.scss:353]: Fixed

File: src/index.scss [hunk 3]
```diff
@@ -496,6 +504,10 @@ header {
   z-index: 1;
 }
 
+.comment-line {
+  width: calc(100% - 180px);
```
[the remark this answers is not in the document]
Comment (vladislavkeblysh) [reply src/index.scss:500]: Fixed

File: src/discussions/common/AuthorLabel.jsx [hunk 4]
```diff
@@ -57,6 +57,7 @@ const AuthorLabel = ({
       })}
       role="heading"
       aria-level="2"
+      style={{ lineHeight: 1, wordBreak: 'break-all' }}
```
[the remark this answers is not in the document]
Comment (vladislavkeblysh) [reply src/discussions/common/AuthorLabel.jsx:60]: Fixed

**Review by awais-ansari — APPROVED** [review]

**Comment by awais-ansari** [comment]
@mphilbrick211 Is this PR still a product review? or is it ready for merge?

**Review by sundasnoreen12 — APPROVED** [review]
PR LGTM

**Comment by mphilbrick211** [comment]
> @mphilbrick211 Is this PR still in a product review? or is it ready for merge?

We haven't heard back from Product - so please hold off on merging.

CC: @jmakowski1123

**Comment by jmakowski1123** [comment]
This is approved.

**Comment by awais-ansari** [comment]
@vladislavkeblysh Please update this PR.

**Comment by vladislavkeblysh** [comment]
@awais-ansari  Done, thank you

**Comment by mphilbrick211** [comment]
All set to merge, @awais-ansari!

**Comment by awais-ansari** [comment]
@vladislavkeblysh can you please update this PR one more time?

**Comment by openedx-webhooks** [comment]
@vladislavkeblysh 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/frontend-app-discussions&entry.867055334=https://github.com/openedx/frontend-app-discussions/pull/577&entry.1484655318=https://github.com/vladislavkeblysh&entry.752974735=2023-10-12+12:35&entry.1917517419=2023-12-18+11:26&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**